src/strategies/views/automations-view.test.ts

```
@@ -0,0 +1,22 @@
+import { describe, expect, test, vi } from 'vitest';
+
+vi.mock('@smolpack/hasskit', () => ({
+  Home: class {
+    floors = [];
+    constructor(_hass: unknown, _config?: unknown) {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)